servlet-feature-pack/src/main/resources/content/docs/licenses/servlet-feature-pack-licenses-addendum.html

```diff
@@ -53,7 +53,7 @@ <h2>WildFly 13 - Servlet Feature Pack - Addendum</h2>
     </td>
     </tr>
     <tr>
-        <td>org.glassfish</td><td>javax.json</td><td>1.0.4</td><td><a href="https://javaee.github.io/glassfish/LICENSE">Common Development and Distribution License 1.1</a>
+        <td>org.glassfish</td><td>javax.json</td><td>1.1.2</td><td><a href="https://javaee.github.io/glassfish/LICENSE">Common Development and Distribution License 1.1</a>
```

Comment:
@jamezp I think 1.1.2 shows up in the normal xxx-pack-licenses.html?

Reply:
It's not really a big deal. These addendum.html files will be gone for WF14 https://issues.jboss.org/browse/WFLY-10643

Reply:
Ah okay. I forgot about that. I should have probably removed it from there.